MvvmCross/Platform/Platform/Converters/MvxDictionaryValueConverter.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace MvvmCross.Platform.Converters
+{
+    public class MvxDictionaryValueConverter<TKey, TValue> : MvxValueConverter<TKey, TValue>
+    {
+        protected override TValue Convert(TKey value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (parameter is Dictionary<TKey, TValue> dict)
+            {
+                TValue x = dict[value];
```

Comment:
We should probably ask here if `dict` contains a key with the appropiate value, and leave a trace in case it doesn't, to make it easier to debug for developers.